packages/xo-web/src/xo-app/logs/backup-ng-logs.js

```diff
@@ -224,25 +257,34 @@ const LOG_RESTORE_COLUMNS = [
     name: _('labelSpeed'),
     itemRenderer: task => {
       const duration = task.end - task.start
-      return duration > 0 && formatSpeed(task.dataSize, duration)
+      return (
+        task.dataSize !== undefined &&
+        duration > 0 &&
+        formatSpeed(task.dataSize, duration)
+      )
     },
     sortCriteria: task => {
       const duration = task.end - task.start
-      return duration > 0 && task.dataSize / duration
+      return (
+        task.dataSize !== undefined && duration > 0 && task.dataSize / duration
+      )
     },
   },
 ]
 
 const ROW_TRANSFORM = (task, { vms }) => {
-  const { id, size: dataSize } =
-    get(
-      () => task.tasks.find(({ message }) => message === 'transfer').result
-    ) || {}
+  let vm, dataSize
+  if (task.status === 'success') {
+    const result = task.tasks.find(({ message }) => message === 'transfer')
```

Comment:
Does it *always* exist? Why did you remove `get`?

Reply:
if it's successful, the transfer task must exists and must contains the vm id and the data size.